src/screens/TopicPageScreen/index.js

```diff
@@ -396,10 +401,17 @@ const TopicPageScreen = (props) => {
     />
   );
 
-  if (isInitialLoading) return null;
   return (
-    <SafeAreaProvider forceInset={{top: 'always'}} style={styles.parentContainer}>
+    <View>
       <NavHeader
+        initialData={{
+          channelPicutre: params.channelPicture,
+          coverImage: params.coverImage,
+          isFollowing: params.isFollowing,
+          memberCount: params.memberCount
+        }}
+        isLoading={isInitialLoading}
+        opacityImage={opacityImage}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes here are significant as they introduce conditional rendering based on the `isInitialLoading` state. This can improve the user experience by providing feedback during loading states. However, it's crucial to ensure that all necessary data is included in the `initialData` prop and that the `isLoading` and `opacityImage` props are correctly managed throughout the component lifecycle.

```diff
+ return (
+   <View>
+      <NavHeader
+       initialData={{
+         channelPicutre: params.channelPicture,
+         coverImage: params.coverImage,
+         isFollowing: params.isFollowing,
+         memberCount: params.memberCount
+       }}
+       isLoading={isInitialLoading}
+       opacityImage={opacityImage}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->